README.rst

```diff
@@ -36,7 +36,14 @@ Installation
 
 Note that if you've already installed `mesos` locally, you can either install this to a location other than `/usr/local/bin` via. pip options or remove `/usr/local/bin/mesos`. There should be no downsides to just removing it.
 
-To build `mesos-cli`, run `python setup.py build` in the root of the repository. To install, run `python setup.py install`.
+
+To build, run ``python setup.py build`` in the root of the repository.
+
+To install, run ``python setup.py install``
+
+To make the development environment, run ``make env && source env/bin/activate``
```

Comment:
I think that makes sense.  This is my first mesos PR, and I was basing it off this example: https://github.com/mesosphere/dcos-helloworld/blob/master/Makefile

But if it doesn't violate some standard you have, I can change it.